stream-chat-android/src/main/java/com/getstream/sdk/chat/viewmodel/channels/ChannelsViewModel.kt

```diff
@@ -86,4 +86,8 @@ public class ChannelsViewModelImpl(
     private fun requestMoreChannels() {
         chatDomain.useCases.queryChannelsLoadMore(filter, sort).enqueue()
     }
+
+    override fun onCleared() {
```

Comment:
What's the reason for non-customized overriding?

Reply:
Was using it for debugging and forgot to remove it!